src/environments/environment.local.ts

```diff
@@ -0,0 +1,12 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=prod` then `environment.prod.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
+  production: false,
+  readBaseUrl: 'http://localhost:8080/admin-api'
```

Comment:
Why is this called `readBaseUrl`? The terminology doesn't make sense to me.

Reply:
Fixed.  Left over from when I thought I would do reads to the main API and writes to an admin API